[28_2_2018_11:21:29] {SEVERE} CONNECTIONDB - java.lang.ClassNotFoundException: com.mysql.cj.jdbc.Driver	at java.net.URLClassLoader.findClass(Unknown Source)
	at java.lang.ClassLoader.loadClass(Unknown Source)
	at sun.misc.Launcher$AppClassLoader.loadClass(Unknown Source)
	at java.lang.ClassLoader.loadClass(Unknown Source)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Unknown Source)
	at org.telegram.plugins.echo.database.ConnectionDB.openConexion(ConnectionDB.java:30)
	at org.telegram.plugins.echo.database.ConnectionDB.<init>(ConnectionDB.java:24)
	at org.telegram.plugins.echo.database.DatabaseManagerImpl.<init>(DatabaseManagerImpl.java:32)
	at it.ban.javatelegram.Deepthought.main(Deepthought.java:39)

[28_2_2018_11:21:48] {SEVERE} CONNECTIONDB - java.lang.ClassNotFoundException: com.mysql.cj.jdbc.Driver	at java.net.URLClassLoader.findClass(Unknown Source)
	at java.lang.ClassLoader.loadClass(Unknown Source)
	at sun.misc.Launcher$AppClassLoader.loadClass(Unknown Source)
	at java.lang.ClassLoader.loadClass(Unknown Source)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Unknown Source)
	at org.telegram.plugins.echo.database.ConnectionDB.openConexion(ConnectionDB.java:30)
	at org.telegram.plugins.echo.database.ConnectionDB.<init>(ConnectionDB.java:24)
	at org.telegram.plugins.echo.database.DatabaseManagerImpl.<init>(DatabaseManagerImpl.java:32)
	at it.ban.javatelegram.Deepthought.main(Deepthought.java:39)

